scripts/test/lint.py

```diff
@@ -273,6 +273,41 @@ def check(self, path):
         folder = path.parts[-2]
 
         return not (pkg_id == folder == nuspec[:-len(".nuspec")])
+        
+        
+        
+class UsesInvalidCategory(Lint):
+    # The common.vm, debloat.vm, and installer.vm packages are special as they
+    # assist with the installation and allow to share code between packages.
+    # They do not install any tool and consequently don't have a category.
+    # excluded dcode because it fails to be installed
```

Comment:
you forgot to remove this line (that you ave moved to just before the dcode exclusion)
```suggestion
```